src/routes/settings/assistants/[assistantId]/+page.server.ts

```diff
@@ -71,6 +73,31 @@ export const actions: Actions = {
 		if (!acknowledged) {
 			return fail(500, { error: true, message: "Failed to report assistant" });
 		}
+
+		if (WEBHOOK_URL_REPORT_ASSISTANT) {
+			const prefixUrl = PUBLIC_SHARE_PREFIX || `${PUBLIC_ORIGIN || url.origin}${base}`;
+			const assistantUrl = `${prefixUrl}/assistant/${params.assistantId}`;
+
+			const assistant = await collections.assistants.findOne(
+				{ _id: new ObjectId(params.assistantId) },
+				{ projection: { name: 1 } }
+			);
+
+			const res = await fetch(WEBHOOK_URL_REPORT_ASSISTANT, {
+				method: "POST",
+				headers: {
+					"Content-type": "application/json",
+				},
+				body: JSON.stringify({
+					text: `Assistant <${assistantUrl}|${assistant?.name}> reported by <http://hf.co/${locals.user?.username}|${locals.user?.username}>`,
```

Comment:
[this is slack msg template for links](https://api.slack.com/reference/surfaces/formatting#linking-urls)